Jenkinsfile

```diff
@@ -78,7 +81,7 @@ def rpm_test_daos_test = '''me=\\\$(whoami)
                                 echo \"Server stdout: \\\$line\"
                             done
                             echo \"Server started!\"
-                            daos_agent &
+                            daos_agent -i &
```

Comment:
Is certificate handling being tested elsewhere?

Reply:
There are no functional tests for this code yet. Ding and I are working on the security validation plan which will include the tests for these. The certificate handling code is handled in the go unit tests.